algorithms/sort/bubble_sort.py

```diff
@@ -20,10 +20,11 @@ def swap(i, j):
     iteration = 0
     if simulation:
         print("iteration",iteration,":",*arr)
-    
+    x=-1
```

Comment:
please use spaces before and after `=` following PEP guide.

Reply:
Okay!